app/src/i18n/locales/es/dashboard.json

```diff
@@ -1,200 +1,200 @@
-  "methodology-and-assumptions-description": "El modelo de pronóstico de emisiones sin acción para Brasil asume una fuerte correlación directa entre las emisiones y el crecimiento económico, reflejando el estado actual de Brasil donde la expansión económica y las emisiones aún no se han desacoplado.",
-  "download-and-report": "Descargar y reportar",
-  "invite-collaborators": "Invitar colaboradores",
-  "invite-your-colleagues": "Invita a tus colegas",
-  "select-cities-to-share": "Selecciona las ciudades para compartir",
-  "select-cities-to-share-description": "Cada usuario que invites recibirá un enlace por ciudad para agregar cada una. Tendrán acceso a todos los inventarios realizados en cada ciudad.",
-  "send-invites": "Enviar invitaciones",
-  "send-invites-description-1": "Ten en cuenta que cada miembro del equipo que invites podrá ",
-  "send-invites-description-2": "agregar, editar y eliminar",
-  "send-invites-description-3": " cualquier inventario al que tengan acceso.",
-  "email": "Correo electrónico",
-  "invalid-email": "Introduce un correo electrónico válido",
-  "email-already-exists": "Correo electrónico duplicado",
-  "press-enter-to-add": "Presiona enter para agregar",
-  "invite-success-toast-title": "Invitación enviada",
-  "invite-success-toast-description": "Tus invitaciones se han enviado con éxito",
-  "invite-success-toast-text": "¡Todo listo! Tus invitaciones se han enviado con éxito",
-  "invite-error-toast-title": "Algo salió mal",
-  "invite-error-toast-description": "Hubo un error al enviar tus invitaciones",
-  "invite-error-toast-text": "Hubo un error al enviar tus invitaciones",
-  "afolu-other-agriculture": "Otra agricultura",
-  "afolu-livestock": "Ganado",
-  "afolu-land": "Tierra",
-  "manage-missing-subsectors": "Gestiona los subsectores faltantes",
-  "rate": "tasa",
-  "sector": "sector",
-  "an-error-occurred": "Ocurrió un error",
-  "error-fetching-sector-breakdown": "Error al obtener el desglose por sector",
-  "source-open-climate": "Fuente: OpenClimate",
-  "projects": "Proyectos",
-  "n-a": "N/D",
-  "project": "Proyecto",
-  "select-project": "Selecciona un Proyecto",
-  "select-project-to-invite-collaborators": "Selecciona el proyecto para invitar a colaboradores",
-  "project-to-share": "Selecciona qué proyecto quieres compartir",
-  "upgrade-plan-city-limit-heading": "¿Necesitas actualizar tu plan?",
-  "upgrade-plan-city-limit-text": "Parece que has alcanzado el límite de ciudades que se pueden agregar a este proyecto. Para actualizar tu plan, por favor contacta a <mail>info@openearth.com</mail>.",
-  "tab-cap-title": "Priorización de la acción climática",
-  "dashboard": "Tablero",
-  "about": "Acerca de",
-  "collaborators": "Colaboradores",
-  "default-project": "Proyecto predeterminado",
-  "go-to": "Ve a...",
-  "industrial-processes-and-product-uses-ippu": "IPPU",
-  "agriculture-forestry-and-other-land-use-afolu": "AFOLU"
-}
+    "welcome-back": "Bienvenido de nuevo",
```

Comment:
Can you revert the indentation changes in this file? The other JSON files are also indented with 2 spaces it seems and it's helpful to keep the changes in these files to a minimum to prevent merge conflicts are they are frequently updated by PRs.